main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import "fmt"
+
+func worker(id int, jobs <-chan int, results chan<- int) {
+	for j := range jobs {
+		go func() {
+			switch j % 3 {
+			case 0:
+				j = j * 1
+			case 1:
+				j = j * 2
+				results <- j * 2
+			case 2:
+				results <- j * 3
```

Comment:
I know this is just a code test, but the variable names are so abstract!
using `i`, `j`, `w` for small cicles it's okay but this can be a problem with complex code blocks
also the `jobs` and `jobs2` naming can be confusing for maintainment purposes